src/sass/_variables.scss

```diff
@@ -0,0 +1,54 @@
+
+$theme-colors: (
+
+    'grey'              : #eeeeee,
```

Comment:
THe intent to organise colors this way is good. I would suggest to keep separate file for colors, separate for fonts etc.

Reply:
Thanks for suggestion. I have created a separate file for colors.